docs/current/sdk/go/guides/768421-go-ci.md

```diff
@@ -0,0 +1,49 @@
+---
+slug: /768421/go-ci
+displayed_sidebar: "current"
+---
+
+# Dagger Go SDK in CI
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
+
+<Tabs defaultValue="github-actions"
+groupId="ci-environment"
+values={[
+{label: 'GitHub Actions', value: 'github-actions'},
+{label: 'CircleCI', value: 'circleci'},
+{label: 'GitLab', value: 'gitlab'},
+{label: 'Jenkins', value: 'jenkins'},
+]}>
+
+<TabItem value="github-actions">
+
+```yaml title=".github/workflows/dagger.yml" file=../snippets/go-ci/actions.yml
+```
+
+</TabItem>
+
+<TabItem value="circleci">
+
+```yaml title=".circleci/config.yml" file=../snippets/go-ci/circle.yml
+```
+
+</TabItem>
+
+<TabItem value="gitlab">
+
+```yaml title=".gitlab-ci.yml" file=../snippets/go-ci/gitlab.yml
+```
+
+</TabItem>
+
+<TabItem value="jenkins">
+
+```groovy title="Jenkinsfile" file=../snippets/go-ci/Jenkinsfile
+```
+
+Requires `docker` client and `go` installed on your Jenkins agent, a Docker host available (can be `docker:dind`), and agents labeled in Jenkins with `dagger`.
```

Comment:
Would it be better to place this and other notes above the code block? My concern is that a user will copy-paste the code block without realizing there are additional requirements when this is placed below.

Reply:
Leaving in place as is for visual harmony, but can change later if needed.